### DISCUSSION TITLE
Unale to install nocodb in arm64 [Jun 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6003] [Discord Threads]
> 🙋‍♂️ **r4nchy**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### can you verify if `nocodb` has been downloaded to your current directory?  
#  
> 🙋‍♂️ **r4nchy**  
#### yes the binary file is present in the current directory  
#  
> 🙋‍♂️ **r4nchy**  
#### i am running it on termux i thought i should mention that  
#  
> 🙋‍♂️ **wkw**  
#### if the file s here, then u probably hv storage permission or u r in the wrong directory  
#  
> 🙋‍♂️ **r4nchy**  
#### I think I am in the correct directory 🙂  
![image0](https://cdn.discordapp.com/attachments/1109725350239424512/1110567909048209469/7d6GAbz5wJ.png)  
#  
> 🙋‍♂️ **r4nchy**  
#### but what permission does it require ?  
#  
> 🙋‍♂️ **r4nchy**  
#### this is the permission of the `noco` dir   
#### ```   
#### drwxrw-rw-  2 u0_a252 u0_a252     3488 May 23 19:15 noco   
#### ```  
#  
> 🙋‍♂️ **wkw**  
#### havent used termux tho - can u help run the following command and show us the result   
####    
#### ```   
#### ldd nocodb   
#### ```  
#  
> 🙋‍♂️ **r4nchy**  
####   
![image0](https://cdn.discordapp.com/attachments/1109725350239424512/1111408690814263306/Iit6BIomzx.png)  
#  
> 🙋‍♂️ **r4nchy**  
#### I get this output  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1109725350239424512)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
